src/components/shortcuts/shortcutUtil.ts

```diff
@@ -0,0 +1,10 @@
+export default function debounce<T extends (...args: any[]) => any>(
```

Comment:
please just use the debounce library instead of defining this function. you will see in other parts of the codebase examples of debounce library